src/docs/software/using/spark.md

```diff
@@ -46,7 +46,7 @@ task within that allocation:
 #SBATCH --nodes=1
 #SBATCH --cpus-per-task=8
 #SBATCH --time=10
-
+module load java
```

Comment:
The `spark`module will automatically load `java` as a dependency (or more precisely, `spark` depends on `hadoop`, which depends on `java`). So it's not necessary to load `java` explicitly.
So:
```
$ ml reset
$ ml spark
$ ml av java

--------------------------------------------- devel -- compilers, MPI, languages, libs ----------------------------------------------
   java/1.8.0_131 (L)

  Where:
   L:  Module is loaded
```